server/startup.js.coffee

```diff
@@ -10,15 +10,20 @@ Meteor.startup ->
   if Releases.find().count() is 0
     Releases.insert
       squad: 'Front End'
-      name: '4.7'
+      name: 'FE4.7'
       releaseDate: new Date('03/31/2015')
 
     Releases.insert
       squad: 'Platform'
-      name: '5.0'
+      name: 'P4.7'
+      releaseDate: new Date('03/31/2015')
```

Comment:
Might as well update the 4.7 release dates while we're at it - let's just set it to '05/08/2015' for now to match the Chromecasted version.